.github/workflows/cherryPick.yml

```diff
@@ -40,17 +40,12 @@ jobs:
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
         with:
           ref: staging
-          token: ${{ secrets.OS_BOTIFY_TOKEN }}
```

Comment:
We used to pass this token to try and set up git as OSBotify. I don't think it ever really worked, but in any event it's not necessary because in the next step we call `setupGitForOSBotify`.